.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -1,78 +1,68 @@
 ---
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
 name: Bug report
-
 about: Report an issue with the current behavior of Traffic Control
-
 labels: bug
-
 ---
-
 <!--
 ************ STOP!! ************
 If this issue identifies a security vulnerability, DO NOT submit it! Instead, contact
 the Apache Traffic Control Security Team at security@trafficcontrol.apache.org and follow the
-guidelines at https://www.apache.org/security/ regarding vulnerability disclosure.
+guidelines at https://apache.org/security regarding vulnerability disclosure.
 
 - For *SUPPORT QUESTIONS*, use the #traffic-control channel on the ASF slack (https://s.apache.org/slack-invite)
-or Traffic Control mailing lists (https://trafficcontrol.apache.org/mailing_lists).
-- Before submitting, please **SEARCH GITHUB** for a similar issue or PR.
+or the Traffic Control Users mailing list (send an email to users-subscribe@trafficcontrol.apache.org to subscribe).
+- Before submitting, please **SEARCH GITHUB** for a similar issue or PR
+    * https://github.com/apache/trafficcontrol/issues
+    * https://github.com/apache/trafficcontrol/pulls
 -->
 
 ## I'm submitting a ...
```

Comment:
i'm wondering if this should just say "I'm submitting a bug report which i consider to have a severity of:"

```
- minor
- medium
- major
- critical
```

would be nice to get some guidance on what level of severity they consider the bug to be...

Reply:
Issue triage is something for maintainers of the project to handle, users shouldn't have to answer an additional question.

Changed *I'm submitting a ...* to a *Bug report* title in 6f12ee4525.

Reply:
yeah, i was just thinking it would be nice what their opinion was. 

```
- minor
- medium
- major
- critical
- not sure
```

but i'm fine leaving that to project maintainers.

Reply:
Yeah, severity should really be relative to the existing severities that have been assigned to other issues by the project maintainers. Otherwise, severities will be all over the place. One person's major is another person's minor.